my_list.rb

```diff
@@ -0,0 +1,16 @@
+require './MyEnumerable'
```

Comment:
- [ ] There is a bug associated with loading the `my_enumerable` module because you have got the name of the file wrong in this line of code. Please fix this error 👍 
![image](https://user-images.githubusercontent.com/46839548/209409874-50ac8c9e-0590-4e29-80b6-a854bf568bd0.png)